test/http_close_test.dart

```diff
@@ -0,0 +1,153 @@
+// Copyright (c) 2013, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import "dart:async";
+import "dart:math";
+import "dart:typed_data";
+
+import "package:http_io/http_io.dart";
+import "package:test/test.dart";
+
+Future<Null> testClientAndServerCloseNoListen(int connections) {
+  Completer<Null> completer = new Completer();
```

Comment:
I think you need to say `new Completer<Null>()` for strong-mode.

Reply:
The analyzer is happy, so I'm going to leave this as-is. If there is a runtime strong-mode problem, I'll change all instances of this all in one go.